transport/UdpEndpoint.cpp

```diff
@@ -52,9 +52,9 @@ UdpEndpoint::UdpEndpoint(jobmanager::JobManager& jobManager,
     RtcePoll& epoll,
     bool isShared)
     : BaseUdpEndpoint("UdpEndpoint", jobManager, maxSessionCount, allocator, localPort, epoll, isShared),
-      _iceListeners(maxSessionCount),
-      _dtlsListeners(maxSessionCount * 8),
-      _iceResponseListeners(maxSessionCount * 4)
+      _iceListeners(maxSessionCount * 2),
```

Comment:
We need more slots in the map than max expected otherrwise it will be inefficient with many hash collisions.